jumpscale/packages/backup/actors/threebot_deployer.py

```diff
@@ -22,6 +22,13 @@ def downloaded_packages_path(self):
             j.sals.fs.mkdirs(packages_path)
         return packages_path
 
+    @property
+    def ssh_dir_path(self):
+        ssh_path = j.sals.fs.join_paths(j.core.dirs.HOMEDIR, ".ssh")
```

Comment:
should be `/root`/ simply because j.core.dirs.HOMEDIR doesn't need to be exactly the same as on the machine running 3bot deployer